syne_tune/optimizer/schedulers/searchers/random_searcher.py

```diff
@@ -0,0 +1,38 @@
+import logging
+from typing import Optional, List, Dict, Any
+
+from syne_tune.optimizer.schedulers.searchers.searcher import BaseSearcher
+
+logger = logging.getLogger(__name__)
+
+
+class RandomSearcher(BaseSearcher):
+    """
+    Searcher which randomly samples configurations to try next.
+    """
+
+    def __init__(
+        self,
+        config_space: Dict[str, Any],
+        points_to_evaluate: Optional[List[dict]] = None,
+        random_seed: int = None,
```

Comment:
The correct type would be:
```suggestion
        random_seed: int | None = None,
```